src/VuFindHarvest/ResponseProcessor/SimpleXmlResponseProcessor.php

```diff
@@ -93,8 +102,8 @@ protected function logBadXML($xml)
     protected function sanitizeXml($xml)
     {
         // Sanitize the XML if requested:
-        $regex = '/[^\x{0009}\x{000a}\x{000d}\x{0020}-\x{D7FF}\x{E000}-\x{FFFD}]+/u';
-        $newXML = trim(preg_replace($regex, ' ', $xml, -1, $count));
+        $this->regex[] = '/[^\x{0009}\x{000a}\x{000d}\x{0020}-\x{D7FF}\x{E000}-\x{FFFD}]+/u';
```

Comment:
Rather than forcing this regex to always be included, would it be better to make this the default that gets set in the constructor if the setting is missing?